2024-10-08 11:10:37,262 INFO    StreamThr :22136 [internal.py:wandb_internal():85] W&B internal server running at pid: 22136, started at: 2024-10-08 11:10:37.260402
2024-10-08 11:10:37,269 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status
2024-10-08 11:10:37,274 INFO    WriterThread:22136 [datastore.py:open_for_write():87] open: .\wandb\run-20241008_111037-ydiihuny\run-ydiihuny.wandb
2024-10-08 11:10:37,280 DEBUG   SenderThread:22136 [sender.py:send():379] send: header
2024-10-08 11:10:37,280 DEBUG   SenderThread:22136 [sender.py:send():379] send: run
2024-10-08 11:10:37,287 INFO    SenderThread:22136 [sender.py:_setup_resume():749] checking resume status for None/rl4co/ydiihuny
2024-10-08 11:10:39,151 INFO    SenderThread:22136 [dir_watcher.py:__init__():211] watching files in: .\wandb\run-20241008_111037-ydiihuny\files
2024-10-08 11:10:39,152 INFO    SenderThread:22136 [sender.py:_start_run_threads():1188] run started: ydiihuny with start time 1728357037.2634
2024-10-08 11:10:39,174 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: check_version
2024-10-08 11:10:39,174 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: check_version
2024-10-08 11:10:40,902 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: run_start
2024-10-08 11:10:41,539 DEBUG   HandlerThread:22136 [system_info.py:__init__():26] System info init
2024-10-08 11:10:41,539 DEBUG   HandlerThread:22136 [system_info.py:__init__():41] System info init done
2024-10-08 11:10:41,539 INFO    HandlerThread:22136 [system_monitor.py:start():194] Starting system monitor
2024-10-08 11:10:41,541 INFO    HandlerThread:22136 [system_monitor.py:probe():214] Collecting system info
2024-10-08 11:10:41,542 INFO    SystemMonitor:22136 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-10-08 11:10:41,544 INFO    SystemMonitor:22136 [interfaces.py:start():188] Started cpu monitoring
2024-10-08 11:10:41,552 INFO    SystemMonitor:22136 [interfaces.py:start():188] Started disk monitoring
2024-10-08 11:10:41,555 INFO    SystemMonitor:22136 [interfaces.py:start():188] Started gpu monitoring
2024-10-08 11:10:41,558 INFO    SystemMonitor:22136 [interfaces.py:start():188] Started memory monitoring
2024-10-08 11:10:41,560 INFO    SystemMonitor:22136 [interfaces.py:start():188] Started network monitoring
2024-10-08 11:10:41,583 DEBUG   HandlerThread:22136 [system_info.py:probe():152] Probing system
2024-10-08 11:10:41,583 DEBUG   HandlerThread:22136 [system_info.py:probe():200] Probing system done
2024-10-08 11:10:41,583 DEBUG   HandlerThread:22136 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.15', 'heartbeatAt': '2024-10-08T03:10:41.583432', 'startedAt': '2024-10-08T03:10:37.079069', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'E:\\rl4ev\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'DESKTOP-4UKOA1U', 'username': '78788', 'executable': 'd:\\py\\Anaconda3\\envs\\pytorch\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 6, 'cpu_freq': {'current': 2901.0, 'min': 0.0, 'max': 2901.0}, 'cpu_freq_per_core': [{'current': 2901.0, 'min': 0.0, 'max': 2901.0}], 'disk': {'/': {'total': 953.8369140625, 'used': 22.45068359375}}, 'gpu': 'NVIDIA GeForce GTX 1050 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1050 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.857707977294922}}
2024-10-08 11:10:41,583 INFO    HandlerThread:22136 [system_monitor.py:probe():224] Finished collecting system info
2024-10-08 11:10:41,583 INFO    HandlerThread:22136 [system_monitor.py:probe():227] Publishing system info
2024-10-08 11:10:41,584 DEBUG   HandlerThread:22136 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-10-08 11:10:41,589 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:41,589 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:42,194 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_111037-ydiihuny\files\conda-environment.yaml
2024-10-08 11:10:43,839 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:43,840 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:46,141 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:46,141 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:48,441 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:48,441 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:50,746 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:50,746 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:53,052 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:53,053 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:55,359 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:55,359 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:56,600 ERROR   HandlerThread:22136 [system_info.py:_save_conda():223] Error saving conda packages: Command '['conda', 'env', 'export']' timed out after 15 seconds
Traceback (most recent call last):
  File "d:\py\Anaconda3\envs\pytorch\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 216, in _save_conda
    subprocess.call(
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 351, in call
    return p.wait(timeout=timeout)
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 1189, in wait
    return self._wait(timeout=timeout)
  File "d:\py\Anaconda3\envs\pytorch\lib\subprocess.py", line 1473, in _wait
    raise TimeoutExpired(self.args, timeout)
subprocess.TimeoutExpired: Command '['conda', 'env', 'export']' timed out after 15 seconds
2024-10-08 11:10:56,603 DEBUG   HandlerThread:22136 [system_info.py:_save_conda():224] Saving conda packages done
2024-10-08 11:10:56,608 INFO    HandlerThread:22136 [system_monitor.py:probe():229] Finished publishing system info
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:10:56,620 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: keepalive
2024-10-08 11:10:56,622 DEBUG   SenderThread:22136 [sender.py:send():379] send: files
2024-10-08 11:10:56,622 INFO    SenderThread:22136 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-10-08 11:10:57,282 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_111037-ydiihuny\files\wandb-metadata.json
2024-10-08 11:10:57,600 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: python_packages
2024-10-08 11:10:57,600 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: python_packages
2024-10-08 11:10:57,604 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:10:57,605 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:10:57,614 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:10:57,659 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:57,660 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:10:58,147 DEBUG   SenderThread:22136 [sender.py:send():379] send: telemetry
2024-10-08 11:10:58,148 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:10:58,149 DEBUG   SenderThread:22136 [sender.py:send():379] send: telemetry
2024-10-08 11:10:58,149 DEBUG   SenderThread:22136 [sender.py:send():379] send: metric
2024-10-08 11:10:58,150 DEBUG   SenderThread:22136 [sender.py:send():379] send: telemetry
2024-10-08 11:10:58,150 DEBUG   SenderThread:22136 [sender.py:send():379] send: metric
2024-10-08 11:10:58,151 WARNING SenderThread:22136 [sender.py:send_metric():1405] Seen metric with glob (shouldn't happen)
2024-10-08 11:10:58,472 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_111037-ydiihuny\files\requirements.txt
2024-10-08 11:10:58,528 INFO    wandb-upload_0:22136 [upload_job.py:push():130] Uploaded file C:\Users\78788\AppData\Local\Temp\tmp1904_hh5wandb\grsot8tu-wandb-metadata.json
2024-10-08 11:10:58,657 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:10:59,667 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:00,261 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:00,261 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:00,684 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:01,697 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:02,552 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:02,552 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:02,712 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:03,632 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:03,738 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:04,541 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_modified():288] file/dir modified: .\wandb\run-20241008_111037-ydiihuny\files\conda-environment.yaml
2024-10-08 11:11:04,743 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:04,853 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:04,854 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:05,755 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:06,770 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:07,169 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:07,169 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:07,773 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:08,658 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:08,782 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:09,473 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:09,474 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:09,598 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_modified():288] file/dir modified: .\wandb\run-20241008_111037-ydiihuny\files\config.yaml
2024-10-08 11:11:09,797 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:10,813 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:11,772 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:11,772 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:11,889 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:12,676 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:11:12,676 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:11:12,953 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:13,651 INFO    Thread-17 :22136 [dir_watcher.py:_on_file_created():271] file/dir created: .\wandb\run-20241008_111037-ydiihuny\files\output.log
2024-10-08 11:11:13,960 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:14,064 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:14,065 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:14,543 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:14,964 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:15,967 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:16,397 ERROR   gpu       :22136 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 11:11:16,408 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:16,409 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:16,978 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:17,992 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:18,660 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:18,661 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:18,999 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:19,712 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:20,012 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:20,969 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:20,970 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:21,026 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:22,039 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:23,050 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:23,278 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:23,279 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:24,054 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:25,073 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:25,585 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:25,586 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:25,715 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:26,078 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:27,093 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:27,669 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:11:27,670 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:11:27,906 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:27,906 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:28,109 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:29,123 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:30,138 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:30,212 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:30,212 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:31,138 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:31,154 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:32,167 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:32,518 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:32,518 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:33,177 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:34,174 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:34,844 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:34,845 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:35,190 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:36,193 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:36,757 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:37,159 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:37,159 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:37,193 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:38,196 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:39,212 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:39,466 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:39,467 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:40,223 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:41,212 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:41,566 DEBUG   SystemMonitor:22136 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-10-08 11:11:41,576 DEBUG   SenderThread:22136 [sender.py:send():379] send: stats
2024-10-08 11:11:41,786 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:41,787 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:42,220 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:42,580 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:42,674 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:11:42,675 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:11:43,235 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:44,108 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:44,109 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:44,250 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:45,253 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:46,257 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:46,424 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:46,425 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:47,273 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:47,711 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:48,289 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:48,728 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:48,728 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:49,301 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:50,313 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:51,077 ERROR   gpu       :22136 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 11:11:51,088 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:51,089 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:51,328 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:52,342 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:52,718 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:53,351 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:53,352 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:53,353 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:54,365 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:55,368 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:55,660 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:55,661 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:56,382 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:57,388 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:57,686 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:11:57,687 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:11:57,981 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:57,981 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:11:58,040 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:11:58,393 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:11:59,402 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:00,296 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:00,296 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:00,413 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:01,430 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:02,443 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:02,591 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:02,591 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:03,457 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:03,754 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:04,473 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:04,887 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:04,888 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:05,473 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:06,482 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:07,208 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:07,208 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:07,498 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:08,514 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:08,763 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:09,519 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:09,520 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:09,528 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:10,544 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:11,559 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:11,592 DEBUG   SenderThread:22136 [sender.py:send():379] send: stats
2024-10-08 11:12:11,815 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:11,816 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:12,567 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:12,692 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:12:12,693 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:12:13,568 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:14,120 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:14,121 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:14,170 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:14,562 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:15,580 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:16,443 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:16,443 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:16,592 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:17,608 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:18,624 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:18,759 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:18,760 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:19,629 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:19,766 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:20,637 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:21,061 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:21,062 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:21,653 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:22,665 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:23,377 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:23,378 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:23,681 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:24,695 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:24,788 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:25,718 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:25,718 ERROR   gpu       :22136 [interfaces.py:aggregate():159] Failed to serialize metric: division by zero
2024-10-08 11:12:25,730 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:25,730 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:26,732 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:27,695 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:12:27,696 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:12:27,745 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:28,048 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:28,048 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:28,757 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:29,769 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:30,079 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:30,377 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:30,377 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:30,784 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:31,780 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:32,686 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:32,687 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:32,788 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:33,808 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:34,829 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:35,010 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:35,011 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:35,817 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:35,864 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:36,830 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:37,302 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:37,302 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:37,832 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:38,825 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:39,592 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:39,592 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:39,835 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:40,822 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:40,840 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:41,601 DEBUG   SenderThread:22136 [sender.py:send():379] send: stats
2024-10-08 11:12:41,849 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:41,886 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:41,886 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:42,720 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: stop_status
2024-10-08 11:12:42,721 DEBUG   SenderThread:22136 [sender.py:send_request():406] send_request: stop_status
2024-10-08 11:12:42,845 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:43,843 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:44,179 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:44,180 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:44,850 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:45,869 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:46,145 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:46,481 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:46,482 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:46,879 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:47,872 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:48,781 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:48,782 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:48,882 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:49,888 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:50,891 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:51,110 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:51,110 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:51,867 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:51,899 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:52,914 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:53,405 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:53,406 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:53,916 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:54,928 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:55,694 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:55,695 ERROR   gpu       :22136 [interfaces.py:monitor():142] Failed to sample metric: Not Supported
2024-10-08 11:12:55,940 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
2024-10-08 11:12:56,887 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: status_report
2024-10-08 11:12:56,949 DEBUG   HandlerThread:22136 [handler.py:handle_request():158] handle_request: internal_messages
